packages/flutter/lib/src/material/debug.dart

```diff
@@ -145,17 +109,15 @@ bool debugCheckHasMaterialLocalizations(BuildContext context) {
 bool debugCheckHasScaffold(BuildContext context) {
   assert(() {
     if (context.widget is! Scaffold && context.ancestorWidgetOfExactType(Scaffold) == null) {
-      final Element element = context;
-      throw FlutterError(
-          'No Scaffold widget found.\n'
-          '${context.widget.runtimeType} widgets require a Scaffold widget ancestor.\n'
-          'The Specific widget that could not find a Scaffold ancestor was:\n'
-          '  ${context.widget}\n'
-          'The ownership chain for the affected widget is:\n'
-          '  ${element.debugGetCreatorChain(10)}\n'
+      throw FlutterError.fromParts(<DiagnosticsNode>[
+        ErrorSummary('No Scaffold widget found.'),
+        ErrorDescription('${context.widget.runtimeType} widgets require a Scaffold widget ancestor.'),
+        ...context.describeMissingAncestor(expectedAncestorType: Scaffold),
```

Comment:
@munificent being able to use `...` made this code a lot more pleasant to write!